src/frontends/ir/src/ir_deserializer.cpp

```diff
@@ -681,29 +695,30 @@ void ov::XmlDeserializer::read_meta_data(const std::shared_ptr<ov::Model>& model
 void ov::XmlDeserializer::read_legacy_meta_data(const std::shared_ptr<ov::Model>& model,
                                                 const std::unordered_set<std::string>& names,
                                                 const pugi::xml_node& root_section) {
-    const auto& read_meta = [](const std::shared_ptr<ov::Model>& model,
-                               const std::string& name,
-                               const pugi::xml_node& meta_section) {
-        auto& rt_info = model->get_rt_info();
-        if (name == "meta_data") {
-            for (const auto& data : meta_section.children()) {
-                const std::string& section_name = data.name();
-                // Rename cli_parameters to conversion_parameters
-                if (section_name == "cli_parameters") {
-                    std::shared_ptr<ov::Meta> meta = std::make_shared<MetaDataParser>("cli_parameters", data);
-                    rt_info["conversion_parameters"] = meta;
-                } else if (!data.attribute("value").empty()) {
-                    rt_info[data.name()] = pugixml::get_str_attr(data, "value");
-                } else {
-                    OPENVINO_THROW("Unsupported legacy argument: ", data.name());
+    const auto& read_meta =
```

Comment:
is it  just indentation change?
if yes, could you please revert?

Reply:
This is odd. I used clang-format-15 as CI uses. But it changed lines, which looks OK for CI